Sources/Models/Object.swift

```diff
@@ -20,7 +20,7 @@ public struct Object {
     
     /// Object description represented as plain json string. 
     /// Returns nil if object cant be represented as json string.
-    var description: String? {
+    public var description: String? {
```

Comment:
I remember efforts to keep this non-public but I guess it's ok (you've used it in the demo app).

Reply:
Yes, thats the thing. I could not find any better way to beutify object's presentation. I was using `objectValue.map.description` but it was showing types as well

<img width="307" alt="image" src="https://github.com/TogglesPlatform/Toggles/assets/9494262/dd855030-669b-47ed-9eea-e08588fbdea8">